### DISCUSSION TITLE
Feature Request: Supertag = Properties on Tag

### DISCUSSION THREAD

**Discussion by houshasen** [toeverything/AFFiNE#7379] [Ideas]
### Background
While using other Personal Knowledge Management (PKM) tools like Heptabase and Tana, I have grown to appreciate the property template feature on "Tags" rather than on individual Docs.

### Detail
For instance, if I want to create book objects, I can assign a tag "book" and attach properties such as author and publication date to this tag. Whenever a doc is tagged with "book," it automatically inherits these properties, which can then be filled out individually. To access all book objects, one simply needs to search for the "book" tag.

### Advanced Feature
Although I am unsure if any current app supports this feature in its entirety, combining it with nested tags could enable the creation of "inheritance" objects. For example, a "person" tag could have a nested "person/family member" tag, where the family member tag includes additional property fields.

**Comment by captorxxv** [comment]
Yes, Please this is a must have feature !
